app/config/config.yml

```diff
@@ -2,9 +2,7 @@ imports:
     - { resource: parameters.yml }
     - { resource: security.yml }
     - { resource: @SumoCodersFrameworkErrorBundle/Resources/config/config.yml }
```

Comment:
Uiteindelijk is het van de corebundle minder belangrijk omdat die vast in de framework repo zit. Ik zal het na het weekend anders eens bekijken.